lib/BackgroundJob/ConvertViewColumnsFormat.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\Tables\BackgroundJob;
+
+use OCP\AppFramework\Utility\ITimeFactory;
+use OCP\BackgroundJob\IJobList;
+use OCP\BackgroundJob\Job;
+use OCP\IDBConnection;
+use OCP\IUserManager;
+
+class ConvertViewColumnsFormat extends Job {
```

Comment:
```suggestion
class ConvertViewColumnsFormat extends TimedJob {
	protected int $timeSensitivity = TIME_INSENSITIVE;
```

Mark it time insensitive so that it can be executed in the maintenance window (cf. https://docs.nextcloud.com/server/latest/developer_manual/basics/backgroundjobs.html#heavy-load-and-time-insensitive)